modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/ddl/DdlSqlToCommandConverter.java

```diff
@@ -438,6 +480,61 @@ private DropIndexCommand convertDropIndex(IgniteSqlDropIndex sqlCmd, PlanningCon
         return dropCmd;
     }
 
+    /**
+     * Converts a given CreateZone AST to a CreateZone command.
+     *
+     * @param createZoneNode Root node of the given AST.
+     * @param ctx           Planning context.
+     */
+    private CreateZoneCommand convertCreateZone(IgniteSqlCreateZone createZoneNode, PlanningContext ctx) {
+        CreateZoneCommand createZoneCmd = new CreateZoneCommand();
+
+        createZoneCmd.schemaName(deriveSchemaName(createZoneNode.name(), ctx));
+        createZoneCmd.zoneName(deriveObjectName(createZoneNode.name(), ctx, "zoneName"));
+        createZoneCmd.ifZoneExists(createZoneNode.ifNotExists());
```

Comment:
that looks weird `ifExists = ifNotExists`

Reply:
Fixed, thanks.